Applications/web_portal/web_app/library/source/pages/account_page/submit_button.tsx

```diff
@@ -6,13 +7,65 @@ interface Properties {
   /** Determines what the button's label is. */
   roles: Nexus.AccountRoles;
 
+  /** Determines if the button is disabled.*/
+  isDisabled?: boolean;
+
   /** Indicates the button was clicked. */
   onClick?: () => void;
 }
 
 /** Displays the submission button. */
 export class SubmitButton extends React.Component<Properties> {
+  public static defaultProps = {
+    isDisabled: false,
+    onClick: () => {}
+  };
+  
+  constructor(props: Properties) {
+    super(props);
+    this.buttonText = (() => {
```

Comment:
Remove this field and instead compute the buttonText/label in the render method.